src/redux/books/books.js

```diff
@@ -0,0 +1,40 @@
+const ADD_BOOK = 'ADD_BOOK';
+const DELETE_BOOK = 'DELETE_BOOK';
```

Comment:
Kindly ensure that each action type follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux). Example:
`const ADD_BOOK = bookstore/books/ADD_BOOK`
`const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`

Following this pattern will help you know at a glance which reducer has the action.